lib/cypress_on_rails/configuration.rb

```diff
@@ -2,8 +2,8 @@
 
 module CypressOnRails
   class Configuration
-    attr_accessor :cypress_folder
```

Comment:
this is a breaking change, suggest the method is added again that prints out a deprecation warning and uses install_folder